plugins/mcreator-localization/help/default/block/custom_property_type.md

```diff
@@ -0,0 +1,3 @@
+Defines which type of values can be accepted by this block property:
+* **Logic:** _true_ or _false_
+* **Integer:** Integer numbers in the range from specified minimal value to specified maximal value
```

Comment:
This says specified min/max value, but I am not able to set it anywhere in UI. This will be confusing for the user seeing this message but not being able to set min/max